packages/internal/src/generate/graphqlCodeGen.ts

```diff
@@ -116,7 +117,26 @@ function getLoadDocumentsOptions(filename: string) {
 }
 
 function getPluginConfig() {
-  const pluginConfig: CodegenTypes.PluginConfig = {
+  let prismaModels: Record<string, string> = {}
+  try {
+    // Extract the models from the prisma client and use those to
+    // set up internal redirects for the return values in resolvers.
+    const localPrisma = require('@prisma/client')
```

Comment:
⚠️  making sure this is highlighted - there might be other alternative methods to grabbing the list of prisma models which doesn't involve `require`-ing from the user's node_modules.

Reply:
Thanks for pointing this out - I get your reservation that it feels a little off, but I also think Prisma is designed to do this, with the generated types ending going into node_modules anyway!